src/shared/components/oncoprint/DataUtils.ts

```diff
@@ -283,13 +283,17 @@ export function fillHeatmapTrackDatum<T extends IBaseHeatmapTrackDatum, K extend
 ) {
     trackDatum[featureKey] = featureId;
     trackDatum.study_id = case_.studyId;
-    if (!data || !data.length) {
+
+    // remove data points of which `value` is NaN
+    const dataWithValue =  _.filter(data, (d) => isNaN(d.value));
```

Comment:
I'm confused, wouldn't you want to filter for !isNaN?

Reply:
Eh yes. My bad. I have updated this line to state `!isNaN`.